### PR TITLE
Unify "partitions" and "partition" options

### PR DESCRIPTION
**Pull request by Onager**

### CODE REVIEW

File: plaso/cli/storage_media_tool.py [hunk 1]
```diff
@@ -1003,21 +992,13 @@ def AddStorageMediaImageOptions(self, argument_group):
...
-            'Choose a partition number from a disk image. This partition '
-            'number should correspond to the partition number on the disk '
-            'image, starting from partition 1. All partitions can be '
-            'defined as: "all".'))
-
-    argument_group.add_argument(
-        '--partitions', dest='partitions', action='store', type=str,
-        default=None, help=(
-            'Define partitions that need to be processed. A range of '
+        '--partitions', '--partition', dest='partitions', action='store',
```
Comment (joachimmetz) [inline plaso/cli/storage_media_tool.py]: why not remove partition option?

**Comment by joachimmetz** [comment]
@Onager why not remove the partition option?

**Comment by Onager** [comment]
Because I don't want to have to remember whether the argument is plural or not. This is also why we have arguments with both underscores and hypens, AFAIK.

**Comment by joachimmetz** [comment]
ACK, SG

**Pull request merged**